frontend/src/metabase/components/UserAvatar/UserAvatar.jsx

```diff
@@ -29,7 +29,20 @@ function initial(name) {
 }
 
 function userInitials(user) {
-  return user ? initial(user.first_name) + initial(user.last_name) : null;
+  if (user) {
+    return nameInitials(user) || emailInitials(user);
+  }
+
+  return null;
+}
+
+function nameInitials(user) {
+  return initial(user.first_name) + initial(user.last_name);
+}
+
+function emailInitials(user) {
+  const emailUsername = user.email.split("@")[0];
```

Comment:
Could `email` be null or an empty string?

Reply:
Yeah it's required, afaik it's how we id customers